src/effects/effectrack.cpp

```diff
@@ -309,8 +267,9 @@ OutputEffectRack::OutputEffectRack(EffectsManager* pEffectsManager,
     DEBUG_ASSERT(masterHandleAndGroup != nullptr);
 
     pChainSlot->registerInputChannel(*masterHandleAndGroup);
-    pChain->enableForInputChannel(*masterHandleAndGroup);
-    pChain->setMix(1.0);
+
+    pChainSlot->enableForInputChannel(*masterHandleAndGroup);
+    pChainSlot->setMix(1.0);
```

Comment:
There should not be any public methods of EffectChainSlot for setting enabled input channels or the mix knob. Add a TODO comment noting this and explaining we can resolve this by making these subclasses of EffectChainSlot and removing the EffectRack layer.